mixer/pkg/attribute/stringmap.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 Istio Authors
```

Comment:
I think you need simple unit tests for the list and stringmap types.

Reply:
Yes! I'll add them. Want to make sure coverage is actually working in CI.